src/py/py_multipatch.cpp

```diff
@@ -565,14 +565,14 @@ InterfacesFromBoundaryCenters(const py::array_t<double>& py_center_vertices,
                                      tolerance);
 }
 
-void GetBoundaryOrientation(
+void GetInterfaceOrientation(
     const std::shared_ptr<splinepy::splines::SplinepyBase>& pyspline_start,
     const int& boundary_start,
     const std::shared_ptr<splinepy::splines::SplinepyBase>& pyspline_end,
     const int& boundary_end,
     const double& tolerance,
     int* int_mappings_ptr,
-    bool* bool_orientations_ptr) {
+    int* bool_orientations_ptr) {
```

Comment:
```suggestion
    int* orientations_ptr) {
```
is it bool because you write either 0 or 1?

Reply:
also changed name